docs/releases.rst

```diff
@@ -8,17 +8,12 @@
 tmt-1.38.0
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-A new :ref:`test-runner` section has been added to the tmt
-:ref:`guide`. It describes some important differences between
-running tests on a :ref:`user-system` and scheduling test jobs in
-
-Each execution of ``tmt-report-result`` command inside a shell
-test will now create a tmt subresult. The main result outcome is
-reduced from all subresults outcomes. If ``tmt-report-result`` is
-not called during the test, the shell test framework behavior
-remains the same - the test script exit code still has an impact
-on the main test result. See also
-:ref:`/stories/features/report-result`.
+In order to prevent dangerous commands to be unintentionally run
+on user's system, the :ref:`/plugins/provision/local` provision
+plugin now requires to be executed with the ``--feeling-safe``
+option or with the environment variable ``TMT_FEELING_SAFE`` set
+to ``True``. See the :ref:`/stories/features/feeling-safe` section
+for more details and motivation behind this change.
```

Comment:
I think the check result note should go here once it's merged. It's one of the most important changes.